crates/indexer/src/geyser/accounts/cardinal_token_manager.rs

```diff
@@ -0,0 +1,330 @@
+                    cardinal_token_managers::mint,
+                    cardinal_token_managers::amount,
+                    cardinal_token_managers::kind,
+                    cardinal_token_managers::state,
+                    cardinal_token_managers::state_changed_at,
+                    cardinal_token_managers::invalidation_type,
+                    cardinal_token_managers::recipient_token_account,
+                    cardinal_token_managers::receipt_mint,
+                    cardinal_token_managers::claim_approver,
+                    cardinal_token_managers::transfer_authority,
+                    cardinal_paid_claim_approvers::paid_claim_approver_payment_amount.nullable(),
+                    cardinal_paid_claim_approvers::paid_claim_approver_payment_mint.nullable(),
+                    cardinal_paid_claim_approvers::paid_claim_approver_payment_manager.nullable(),
+                    cardinal_paid_claim_approvers::paid_claim_approver_collector.nullable(),
+                    cardinal_time_invalidators::time_invalidator_address.nullable(),
+                    cardinal_time_invalidators::time_invalidator_payment_manager.nullable(),
+                    cardinal_time_invalidators::time_invalidator_collector.nullable(),
+                    cardinal_time_invalidators::time_invalidator_expiration.nullable(),
+                    cardinal_time_invalidators::time_invalidator_duration_seconds.nullable(),
+                    cardinal_time_invalidators::time_invalidator_extension_payment_amount
+                        .nullable(),
+                    cardinal_time_invalidators::time_invalidator_extension_duration_seconds
+                        .nullable(),
+                    cardinal_time_invalidators::time_invalidator_extension_payment_mint.nullable(),
+                    cardinal_time_invalidators::time_invalidator_max_expiration.nullable(),
+                    cardinal_time_invalidators::time_invalidator_disable_partial_extension
+                        .nullable(),
+                    cardinal_use_invalidators::use_invalidator_address.nullable(),
+                    cardinal_use_invalidators::use_invalidator_payment_manager.nullable(),
+                    cardinal_use_invalidators::use_invalidator_collector.nullable(),
+                    cardinal_use_invalidators::use_invalidator_usages.nullable(),
+                    cardinal_use_invalidators::use_invalidator_use_authority.nullable(),
+                    cardinal_use_invalidators::use_invalidator_total_usages.nullable(),
+                    cardinal_use_invalidators::use_invalidator_extension_payment_amount.nullable(),
+                    cardinal_use_invalidators::use_invalidator_extension_payment_mint.nullable(),
+                    cardinal_use_invalidators::use_invalidator_extension_usages.nullable(),
+                    cardinal_use_invalidators::use_invalidator_max_usages.nullable(),
+                ))
+                .load(db)
```

Comment:
nb: Since you are looking for 1 result you can do `find(1).first(db)` this will return an Err() if the record doesn't exist can check that instead of the len.

https://github.com/diesel-rs/diesel/blob/master/diesel_tests/tests/find.rs#L112